src/Contract/Prelude.purs

```diff
@@ -0,0 +1,302 @@
+-- | A custom Prelude that re-exports Purescript's prelude and further expands.
```

Comment:
Maybe we could add an adaptation of `mconcat` since it's so common in off-chain code